packages/bpk-mixins/src/mixins/_forms.scss

```diff
@@ -108,7 +108,8 @@
 
 @mixin bpk-input--invalid {
   padding-right: $bpk-spacing-base;
-  background: $bpk-color-white url(map-get($bpk-icons-sm, exclamation-circle-red-500)) no-repeat right $bpk-spacing-sm center;
+  border-color: $bpk-color-red-500;
+  background: rgba($bpk-color-red-500, .1) url(map-get($bpk-icons-sm, exclamation-circle-red-500)) no-repeat right $bpk-spacing-sm center;
```

Comment:
I'm not sure we should be actually making this transparent - I think it could have strange side-effects. Could we use red50 or red100 so that it's not actually transparent?
@matthewdavidson @k0nserv wdyt?

Reply:
That's my bad - I assumed transparent would work but I was wrong for the reasons you said above. Good catch 😄

Reply:
Sure I can change it to that :) I also based it off the designs in zeplin that had it at red-500 with .1 opacity, but we can sure try the approach from @georgegillams